functions/README.md

```diff
@@ -9,13 +9,15 @@
 **Details**: Use this extension to:
```

Comment:
I think we had discussed moving this to the root of the repository. This can be done by changing the [`generate-readme` script](https://github.com/mailchimp/Firebase/blob/e9eadacc36bebf02be80ca0263604483d9a86480/functions/package.json#L10) to:

```
firebase ext:info .. --markdown > ../README.md
```

(adding a `../` to the readme path)

Reply:
@jhuleatt Resolved. Must have slipped from last review, thanks for the reminder!